frontend/src/modules/entitieslist/components/Entity.js

```diff
@@ -152,12 +152,16 @@ export default class Entity extends React.Component<Props> {
                         />
                     </p>
                 </div>
-                <button
-                    className='sibling-Entities'
-                    onClick={this.props.getSiblingEntities}
-                >
-                    TEST
-                </button>
+                {search ? (
```

Comment:
You'll need to extend the check to also include filters.

To do that, you'll need to pass the `parameters` prop from `EntityList` to `Entity` and then check if either of the filter parameters (`status`, `extra`, `tag`, `time`, `author`) is set. You could add a helper function, so that the expression when rendering the button will be cleaner, e.g. `search || areFiltersApplied()`.